network/tailscale/tailscale.yaml

```diff
@@ -0,0 +1,73 @@
+name: tailscale
+depends:
+  - service: cri
+  - network:
+    - addresses
+    - connectivity
+    - etcfiles
+container:
+  entrypoint: ./containerboot
+  args: []
+  environment:
+  - PATH=/usr/local/bin:/sbin
+  - TS_SOCKET=/var/run/tailscale/tailscaled.sock
+  - TS_STATE_DIR=/var/lib/tailscale
+  - TS_USERSPACE=false
+  - TS_ACCEPT_DNS=true
+  security:
+    maskedPaths: []
```

Comment:
do we need the `maskedPaths` and `readOnlyPaths` to be non defaults?

Reply:
I am not sure, it was part of debugging. I will try to pair it down

Reply:
Have removed it. Doesn't appear necessary